### DISCUSSION TITLE
Trouble disabling SetRepeatPlayCurrentTrack with MH2024K24SS

### DISCUSSION THREAD

**Discussion by bgberk** [Makuna/DFMiniMp3#111] [Q&A]
Is the library compatibility with Mp3ChipMH2024K16SS the same for MH2024K24SS?

I can SetRepeatPlayCurrentTrack(true) to start looping, but SetRepeatPlayCurrentTrack(false) does not stop tracks from looping. I also have an issue with any get() calls hanging up the dfPlayer, but these aren't necessary in my code. Makes me think that they both might be related to some timing issues discussed with the 16ss compared to the original module.

**Comment by Makuna** [comment]
I personally don't know about the MH2024K24SS.  Sounds like yet another variant that may need special handling.  Did you try the Mp3ChipMH2024K16SS? 

Note this is in reference to my code in their codebase; but seems similar problems.
https://github.com/ghmartin77/DFPlayerAnalyzer/issues/10

The best way to help figure it out is to set the debug flag at the top of a sketch before you include the header, then have that sketch demonstrate a single problem.  Capture the debug out and include that along with sketch and the explanation of what was expected and what happened.
```
#define DfMiniMp3Debug Serial
```

**Comment by Makuna** [comment]
BTW, https://github.com/Makuna/DFMiniMp3/pull/112 just replaced waits for retries and ack requests.  Much more robuts.
